features/nationaliteithistorie/dev/nationaliteit-gegevens.feature

```diff
@@ -0,0 +1,129 @@
+      | B                                  | 310                   |                          | 20010319                        |
+      Als nationaliteithistorie wordt geraadpleegd met de volgende parameters
+      | naam                | waarde                                                                 |
+      | type                | RaadpleegMetPeriode                                                    |
+      | burgerservicenummer | 555550001                                                              |
+      | peildatum           | 2018-01-01                                                             |
+      | fields              | nationaliteiten.redenOpname.code,nationaliteiten.datumIngangGeldigheid |
+      Dan heeft de response de volgende 'nationaliteiten'
+      | type                    | redenOpname.code | datumIngangGeldigheid.type | datumIngangGeldigheid.datum | datumIngangGeldigheid.langFormaat |
+      | BehandeldAlsNederlander | 310              | Datum                      | 2001-03-19                  | 19 maart 2001                     |
+
+    Scenario: persoon heeft een beëindigde nationaliteit met gecorrigeerde reden beëindigen
+      Gegeven de persoon met burgerservicenummer '999994657' heeft een 'nationaliteit' met de volgende gegevens
+      | nationaliteit (05.10) | reden opnemen (63.10) | reden beëindigen (64.10) | datum ingang geldigheid (85.10) |
+      |                       |                       | 404                      | 20140601                        |
+      |                       |                       | 401                      | 19940601                        |
+      | 0100                  | 301                   |                          | 19890301                        |
+      Als nationaliteithistorie wordt geraadpleegd met de volgende parameters
+      | naam                | waarde                                                                                                    |
+      | type                | RaadpleegMetPeriode                                                                                       |
+      | burgerservicenummer | 999994657                                                                                                 |
+      | peildatum           | 2000-01-01                                                                                                |
+      | fields              | nationaliteiten.nationaliteit.code,nationaliteiten.redenOpname.code,nationaliteiten.datumIngangGeldigheid |
+      Dan heeft de response de volgende 'nationaliteiten'
+      | type          | nationaliteit.code | redenOpname.code | datumIngangGeldigheid.type | datumIngangGeldigheid.datum | datumIngangGeldigheid.langFormaat |
+      | Nationaliteit | 0100               | 301              | Datum                      | 1975-05-01                  | 1 mei 1975                        |
+
```

Comment:
En deze scenario kan als volgt worden geschreven:

    Scenario: persoon heeft een beëindigde nationaliteit met gecorrigeerde reden beëindigen
      Gegeven de persoon met burgerservicenummer '999994657' heeft een 'nationaliteit' met de volgende gegevens
      | nationaliteit (05.10) | reden opnemen (63.10) | datum ingang geldigheid (85.10) |
      | 0100                  | 301                   | 19890301                        |
      En de 'nationaliteit' is gewijzigd met de volgende gegevens
      | reden beëindigen (64.10) | datum ingang geldigheid (85.10) |
      | 401                      | 19940601                        |
      En de 'nationaliteit' is vervolgens gewijzigd met de volgende gegevens
      | reden beëindigen (64.10) | datum ingang geldigheid (85.10) |
      | 404                      | 20140601                        |
      Als nationaliteithistorie wordt geraadpleegd met de volgende parameters
      | naam                | waarde                                                                                                    |
      | type                | RaadpleegMetPeriode                                                                                       |
      | burgerservicenummer | 999994657                                                                                                 |
      | peildatum           | 2000-01-01                                                                                                |
      | fields              | nationaliteiten.nationaliteit.code,nationaliteiten.redenOpname.code,nationaliteiten.datumIngangGeldigheid |
      Dan heeft de response de volgende 'nationaliteiten'
      | type          | nationaliteit.code | redenOpname.code | datumIngangGeldigheid.type | datumIngangGeldigheid.datum | datumIngangGeldigheid.langFormaat |
      | Nationaliteit | 0100               | 301              | Datum                      | 1975-05-01                  | 1 mei 1975                        |